bin/dartdoc.dart

```diff
@@ -195,6 +180,11 @@ ArgParser _createArgsParser() {
       allowMultiple: true, help: 'library names to generate docs for');
   parser.addOption('include-external',
       allowMultiple: true, help: 'additional (external) libraries to include');
+  parser.addFlag('include-examples',
+      defaultsTo: false,
+      negatable: false,
+      help:
+          "look for '@example' tags and inject the examples into the comment");
```

Comment:
Add a note for how to find an example? Or, show an example here? Also, how are the relative paths found?